src/main/kotlin/org/rust/lang/core/psi/RustLiteral.kt

```diff
@@ -0,0 +1,8 @@
+package org.rust.lang.core.psi
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.tree.IElementType
+
+interface RustLiteral : PsiElement {
+    val tokenType: IElementType
+}
```

Comment:
Well probably Rust won't introduce literals built of many tokens, so this design could be quite scalable. Will implement this.